plugins/comfyui-installer/scripts/installer.sh

```diff
@@ -0,0 +1,93 @@
+# build system
+apt update
+apt install -y git
+
+# cd to the destination directory
+mkdir -p "$DESTINATION"
+cd "$DESTINATION"
+
+# clone the repository if it doesn't exist. If it does exist, cd into it and update it
+if [ -d "comfyui" ]; then
+  echo "ComfyUI directory already exists. Updating..."
+  cd comfyui
+  git pull origin master
+else
+  echo "Cloning ComfyUI repository..."
+  git clone https://github.com/comfyanonymous/ComfyUI.git --depth 1 comfyui
+  cd comfyui
+fi
+
+# prepare the python environment
+pip install uv
+
+# install python to the system
+# if $DESTINATION/comfyui/py_install/cpython-3.12.11-linux-x86_64-gnu does not exist, then install it
+if [ ! -d "$DESTINATION/comfyui/py_install/cpython-3.12.11-linux-x86_64-gnu" ]; then
+  echo "Installing Python 3.12.11..."
+  uv python install -f -r -i py_install 3.12.11
+else
+  echo "Python 3.12.11 already installed."
+fi
+
+# create the virtual environment if it doesn't exist
+if [ ! -d ".venv" ]; then
+  echo "Creating virtual environment..."
+  echo "Using Python from: $DESTINATION/comfyui/py_install/cpython-3.12.11-linux-x86_64-gnu/bin/python"
+  uv venv .venv -p "$DESTINATION/comfyui/py_install/cpython-3.12.11-linux-x86_64-gnu/bin/python"
+else
+  echo "Virtual environment already exists."
+fi
+source .venv/bin/activate
+uv pip install --no-cache -r requirements.txt
+
+# install comfyui manger
+cd custom_nodes
+
+git config pull.rebase false
+
+# check if the ComfyUI-Manager directory exists, if it does, skip cloning
+if [ -d "comfyui-manager" ]; then
+  echo "ComfyUI-Manager directory already exists..."
+else
+  echo "Cloning ComfyUI-Manager repository..."
+  git clone https://github.com/ltdrdata/ComfyUI-Manager comfyui-manager
+fi
+cd comfyui-manager
+git pull origin main
+uv pip install --no-cache -r requirements.txt
+cd ../
+
+# install comfyui distributed
+
+# check if the ComfyUI-Distributed directory exists, if it does, skip cloning
+rm -rfv ComfyUI-Distributed
+ls -la
+echo "Cloning ComfyUI-Distributed repository..."
+git clone --branch v1.1.0 https://github.com/robertvoy/ComfyUI-Distributed.git
```

Comment:
This is the only real line change. I am pinning to a target release.

The rest of this was simply moved out of the entrypoint.sh into installer.sh to follow our other plugin standards.